notebooks/notebooks/paper/defense-macros.sty

```diff
@@ -0,0 +1,57 @@
+% macros
+% \*short and \*medium model names are used in plots to save space
+\newcommand{\todo}[1]{{\color{red}{#1}}}
+\newcommand{\cpfivezerofivecode}{\texttt{b40-s11841m}}
+\newcommand{\cpfivezerofive}{\texttt{base-victim}}
+\newcommand{\cpfivezerofiveshort}{\texttt{base-v}}
+\newcommand{\bsixtycode}{\texttt{b60c320-s7702m}}
+\newcommand{\bsixty}{\texttt{may23-victim}}
+\newcommand{\origcyclic}{\texttt{base-adversary}}  % go-attack original cyclic \attacker{}
+\newcommand{\origcyclicshort}{\texttt{base-a}}
+\newcommand{\origcyclicmedium}{\texttt{base-adv}}
+\newcommand{\attackbsixty}{\texttt{attack-may23}} % go-attack adv attacking b60-s7702m
+\newcommand{\origtwotwoseven}{\texttt{base-adv-early}} % checkpoint s227m of original cyclic adv
+\newcommand{\dectwentythreecode}{\texttt{b18-s8527m}}
+\newcommand{\dectwentythree}{\texttt{dec23-victim}}
+\newcommand{\dectwentythreeshort}{\texttt{dec23-v}}
+\newcommand{\vitvictim}{\texttt{ViT-victim}}
+\newcommand{\vitvictimshort}{\texttt{ViT-v}}
+\newcommand{\vitadversary}{\texttt{ViT-adversary}}
+\newcommand{\vitadversaryshort}{\texttt{ViT-a}}
+\newcommand{\vitadversarymedium}{\texttt{ViT-adv}}
+\newcommand{\controlbten}{\texttt{control-victim}}
+
+\newcommand{\attackiter}[1]{% attack iteration
+    \ifnum0<0#1\relax
+        \texttt{a\textsubscript{#1}}%
+    \else%
+        \texttt{a\textsubscript{$#1$}}%
+    \fi
+}
+\newcommand{\defenseiter}[1]{% defense iteration
+    \ifnum0<0#1\relax
+        \texttt{v\textsubscript{#1}}%
+    \else%
+        \texttt{v\textsubscript{$#1$}}%
+    \fi
+}
+
+\newcommand{\attackhnine}{\texttt{atari-adversary}} % attack-h9
+\newcommand{\attackhnineshort}{\texttt{atari-a}}
+\newcommand{\contadv}{\texttt{continuous-adversary}} % continuous adversary
+\newcommand{\contadvshort}{\texttt{cont-a}} % continuous adversary
+\newcommand{\contadvmedium}{\texttt{cont-adv}} % continuous adversary
+\newcommand{\koadv}{\texttt{gift-adversary}} % attack-b18
+\newcommand{\koadvshort}{\texttt{gift-a}} % attack-b18
+\newcommand{\koadvmedium}{\texttt{gift-adv}} % attack-b18
+
+\newcommand{\attacker}{adversary}
+\newcommand{\Attacker}{Adversary}
+\newcommand{\attackers}{adversaries}
+\newcommand{\Attackers}{Adversaries}
+\newcommand{\defender}{victim}
+\newcommand{\Defender}{Victim}
+\newcommand{\defenders}{victims}
+\newcommand{\Defenders}{Victims}
+
+\newcommand{\plotdiamond}{$\blacklozenge$}
```

Comment:
my bad on not committing this file in PR #98 — I was developing remote on Flamingo, syncing to local, and pushing from local, but I failed to include this file in the syncing